course_catalog/views.py

```diff
@@ -328,3 +339,40 @@ class TopicViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = CourseTopicSerializer
     pagination_class = LargePagination
     permission_classes = (AnonymousAccessReadonlyPermission,)
+
+
+class WebhookOCWView(APIView):
+    """
+    Handle webhooks coming from the OCW bucket
+    """
+
+    permission_classes = ()
+    authentication_classes = ()
+
+    def handle_exception(self, exc):
+        """Raise any exception with request info instead of returning response with error status/message"""
+        raise WebhookException(
+            "REST Error (%s). BODY: %s, META: %s"
```

Comment:
Can we use an f-string or `.format` here?

Reply:
I think the standard for logging exceptions is to use `%s` - something to do with how sentry handles them, it prevents duplicate issues somehow?

Reply:
Right but this is an exception being raised, not one being logged.